spec/rmagick/image/composite_spec.rb

```diff
@@ -5,15 +5,15 @@
     [
       Magick::UndefinedCompositeOp,
       Magick::NoCompositeOp,
-      Magick::AddCompositeOp,
+      # Magick::AddCompositeOp,
```

Comment:
Were these replaced with something else in IM7?

Reply:
`AddCompositeOp` has been replaces with `ModulusAddCompositeOp` in ImageMagick 7 and `SubtractCompositeOp` has been changed to `ModulusSubtractCompositeOp`. This has also been like this for a while in ImageMagick 6 now. Will need to check what 6.7.7 does.